docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,629 @@
+---
+title: Submitting a pull request to Loopback 4
+lang: en
+keywords: contributing, LoopBack community, pull request, PR, loopback
+sidebar: contrib_sidebar
+permalink: /doc/en/lb4/submitting_a_pr.html
```

Comment:
How about `/doc/en/lb4/submitting_patches.html`?

Reply:
@bajtos , the title is "submitting a pull request", the MD file is "submitting_a_pr.md", so I figured that we should keep the HTML file name consistent with "submitting_a_pr.html" . Do we want to change the title of the document?